src/frontends/pytorch/src/op/sort.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+#define mark(...) context.mark_node(__VA_ARGS__)
+#define shared    std::make_shared
+
+using namespace opset10;
+using namespace ov::op;
+
+OutputVector translate_sort(NodeContext& context) {
```

Comment:
This code is very similar to aten::sort from #15769 Can we merge two implementations?

Reply:
Oki, extracted the common denominator into a sort_elements utils function